MIGRATION.md

```diff
@@ -34,34 +34,45 @@ integrations from the `Integrations.XXX` hash, is deprecated in favor of using t
+| `new TryCatch()`             | `browserApiErrorsIntegration()`     | `@sentry/browser`, `@sentry/deno`                                                                       |
+| `new VueIntegration()`       | `vueIntegration()`                  | `@sentry/vue`                                                                                           |
+| `new DenoContext()`          | `denoContextIntegration()`          | `@sentry/deno`                                                                                          |
+| `new DenoCron()`             | `denoCronIntegration()`             | `@sentry/deno`                                                                                          |
+| `new NormalizePaths()`       | `normalizePathsIntegration()`       | `@sentry/deno`                                                                                          |
+| `new Console()`              | `consoleIntegration()`              | `@sentry/node`                                                                                          |
+| `new Context()`              | `nodeContextIntegration()`          | `@sentry/node`                                                                                          |
+| `new Modules()`              | `modulesIntegration()`              | `@sentry/node`                                                                                          |
+| `new OnUncaughtException()`  | `onUncaughtExceptionIntegration()`  | `@sentry/node`                                                                                          |
+| `new OnUnhandledRejection()` | `onUnhandledRejectionIntegration()` | `@sentry/node`                                                                                          |
+| `new LocalVariables()`       | `localVariablesIntegration()`       | `@sentry/node`                                                                                          |
+| `new Spotlight()`            | `spotlightIntergation()`            | `@sentry/node`                                                                                          |
+| `new Anr()`                  | `anrIntergation()`                  | `@sentry/node`                                                                                          |
+| `new Hapi()`                 | `hapiIntegration()`                 | `@sentry/node`                                                                                          |
+| `new Undici()`               | `nativeNodeFetchIntegration()`      | `@sentry/node`                                                                                          |
+| `new Http()`                 | `httpIntegration()`                 | `@sentry/node`                                                                                          |
```

Comment:
I feel like we need to do a better job to call out that this also applies to all `@sentry/node`-based packages (serverless, fullstack framework SDKs). Doesn't strictly have to happen now but I'd say we should add them. Probably simplest to list them individually.

Reply:
yeah, I thought about this too but this makes the table veeery unwieldly 😓  not sure, maybe we just point it out above? E.g. node also applies to next, remix, sveltekit, ..., browser also applies to react, ...?